packages/help-center/src/hooks/use-get-history-chats.ts

```diff
@@ -73,8 +72,8 @@ export const useGetHistoryChats = (): UseGetHistoryChatsResult => {
 			...conversationsWithUpdatedStatuses,
 			...filteredOdieConversations,
 		].sort( ( a, b ) => {
-			const createdAtA = getConversationCreatedAt( a ) ?? 0;
-			const createdAtB = getConversationCreatedAt( b ) ?? 0;
+			const createdAtA = Number( a.messages?.[ 0 ]?.received ) * 1000 || 0;
+			const createdAtB = Number( b.messages?.[ 0 ]?.received ) * 1000 || 0;
 
 			return createdAtB - createdAtA;
```

Comment:
We should probably rename those variables are their name is no longer accurate:
```suggestion
			const lastMessageReceivedAtA = Number( a.messages?.[ 0 ]?.received ) * 1000 || 0;
			const lastMessageReceivedAtB = Number( b.messages?.[ 0 ]?.received ) * 1000 || 0;

			return lastMessageReceivedAtB - lastMessageReceivedAtA;
```